internal/host/host.go

```diff
@@ -738,14 +738,14 @@ func (m *Manager) reportValidationStatusChanged(ctx context.Context, vc *validat
 		if err := json.Unmarshal([]byte(h.ValidationsInfo), &currentValidationRes); err != nil {
 			return errors.Wrapf(err, "Failed to unmarshal validations info from host %s in cluster %s", h.ID, h.ClusterID)
 		}
-		for vCategory, vRes := range newValidationRes {
+		for vCategory, vRes := range currentValidationRes {
```

Comment:
In case it failed before, I guess we should have UT on this part

Reply:
No unit-tests for metrics, prometheus library doesn't let you get the metric value, only subsystem.